qatrfm/cli.py

```diff
@@ -33,7 +34,27 @@ def print_version(ctx, param, value):
     ctx.exit()
 
 
-CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+def check_image(vars):
+    for var in vars:
+        if (var.split("=")[0] == 'image'):
+            img = var.split("=")[1]
+            if (not os.path.isfile(img)):
+                raise FileNotFoundError("No such image file {}".format(img))
+            return True
+    raise libutils.TrfmMissingVariable(
+        "TF Parameter 'image' must be provided: "
+        "qatrfm ....  --tfvar image=<image_path>")
+
+
+def find_tf_file(basedir):
```

Comment:
This retrieves the first `*.tf` file, should we warn if there exists more then one?

Reply:
for now, it only supports 1 .tf file. If there are more than 1, it will return the first one found. Maybe we can think of providing the tf file as another parameter of the CLI. But better for another PR.

Reply:
Agree, I just think we should warn if we found more then one. But not in this PR.